corehq/apps/app_manager/templates/app_manager/spec/mocha.html

```diff
@@ -3,8 +3,14 @@
 
 {% block dependencies %}
 <script src="{% static 'app_manager/js/form_workflow.js' %}"></script>
+<script src="{% static 'app_manager/js/download_async_modal.js' %}"></script>
 {% endblock %}
 
 {% block mocha_tests %}
 <script src="{% static 'app_manager/spec/form_workflow_spec.js' %}"></script>
+<script src="{% static 'app_manager/spec/download_async_modal_spec.js' %}"></script>
```

Comment:
Question for @benrudolph: why are we naming test files with "spec" instead of "tests"? I've never seen that before.